### PR TITLE
en 

### PR DESCRIPTION
**Pull request by Wen1201**
_If you had no problems you can just leave it empty_

- How difficult did you find this (out of 10)? (0 being no problems at all, 10 being impossible)
- Was there anything that you struggled with?
- Is there any particular code you want me to look into?
- Is there anything that you'd like some further information on?
- Roughly how long did it take?

### CODE REVIEW

**Comment by Wen1201** [comment]
n

File: wen-gong/week5/album-app/app/views/albums/_form.html.erb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    <br><br>
+
+    <%= f.label :image, 'Image URL' %>
+    <br>
+    <%= f.text_field :image, placeholder: 'URL here' %>
+    <br><br>
+
+    <%= f.label :singer_id, 'Singer' %>
+    <%= f.collection_select :singer_id, Singer.all, :id, :name, include_blank: true %>
+
```
Comment (kristabel-wong) [inline wen-gong/week5/album-app/app/views/albums/_form.html.erb:32]: You actually cannot create an album until you select a singer. 

So you can add `required: true` to line 31, if albums MUST be made with a singer_id

Or if you don't need it to be make with the singer id you can add `optional: true` to the Album model (i will show you where)

File: wen-gong/week5/album-app/app/models/album.rb [hunk 2]
```diff
@@ -0,0 +1,4 @@
+class Album < ApplicationRecord
+
+    belongs_to :singer
```
Comment (kristabel-wong) [inline wen-gong/week5/album-app/app/models/album.rb:3]: if you don't need albums to be make with the singer id you can add `optional: true` after `belongs_to :singer`

File: wen-gong/week4/ironman-armor/public/css/style.css [hunk 3]
```diff
@@ -0,0 +1,17 @@
+body {
+    position: relative;
+}
+
```
Comment (kristabel-wong) [inline wen-gong/week4/ironman-armor/public/css/style.css:4]: very hard to read the text - maybe add `color: white` to the file?

File: wen-gong/week4/ironman-armor/public/css/style.css [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+img {
+    position: absolute;
+    z-index: -10;
+    width: 100%;
+    height: 100%;
```
Comment (kristabel-wong) [inline wen-gong/week4/ironman-armor/public/css/style.css:10]: this stretches the image - do `max-width` and `max-height` instead

File: wen-gong/week5/game/config/routes.rb [hunk 5]
```diff
@@ -0,0 +1,12 @@
+Rails.application.routes.draw do
+  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+
+  get '/magic8ball' => 'pages#magic'
+
+  get '/magic8ball/Why do I have to be harassed into submitting homework?' => 'pages#answer'
+
```
Comment (kristabel-wong) [inline wen-gong/week5/game/config/routes.rb:7]: :'(

Comment (kristabel-wong) [reply wen-gong/week5/game/config/routes.rb:7]: Maybe a lesson to not copy some one else's code. ahha 😅

File: wen-gong/week7/homework day1/index.html [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+    <script src="js/jquery.js"></script>
+    <script src="js/main.js"></script>
+
+</head>
+<body>
+
+    <h1>Movie!</h1>
+
+    <input type="text" id="movieQuery" placeholder="Enter a movie">
+    <button id="submitButton">Lookup Number Trivia</button>
```
Comment (kristabel-wong) [inline wen-gong/week7/homework day1/index.html:17]: If you use or copy code from somewhere else, please make sure to update the names.

**Pull request merged**